redmine/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+
+services:
+
+  redmine:
+    image: redmine
+    ports:
+      - 8080:80
```

Comment:
You think I would have noticed the output as well: `WEBrick::HTTPServer#start: pid=1 port=3000`